lte/gateway/c/oai/include/nas/securityDef.h

```diff
@@ -68,6 +68,8 @@ Description Contains global security definitions
 #define AUTH_KENB_SIZE AUTH_KASME_SIZE     /* eNodeB security key   */
 #define AUTH_NEXT_HOP_SIZE AUTH_KASME_SIZE /* Next Hop security parameter*/
 
+#define AUTH_KGNB_SIZE AUTH_KASME_SIZE /* gNodeB security key   */
```

Comment:
There is no KASME in 5G. You can define sizes for KAUS, KSEAF, KAMF and use them.